DTS-STN/Service-Canada-Labs
# Project cards get data from SCLabs-Page-v1 fragments

# Description

Changes query for project cards to fetch from SCLabs-Page-v1 fragments filtered for pages that have a non-null project status.

## Acceptance Criteria

See that project cards are being rendered with the same information before these changes.

## Test Instructions

1. Navigate to projects page
2. See that page and cards render without error
3. Content in cards should be 1:1 with what was in the previous version
